app/controllers/admin/vouchers_controller.rb

```diff
@@ -9,19 +9,31 @@
     end
 
     def create
-      @voucher = Voucher.create(permitted_resource_params.merge(enterprise: @enterprise))
+      voucher_type = params[:voucher][:voucher_type]
+      if Voucher::TYPES.include?(voucher_type)
+        @voucher = voucher_type.safe_constantize.create(
```

Comment:
I think this is safe to ignore, as we are doing whitelist check before calling `safe_constantize`, it's a known brakeman false positing issue : https://github.com/presidentbeef/brakeman/issues/1115

Reply:
I added a comment in the code, seems like the easiest solution. I guess we can look into adding `brakeman.ignore` file if it becomes problematic

Reply:
That's unfortunate. Another solution would be select *our* string based on the params value so that we are not using user input directly. But it's really the same.